File: Day 6 - Lets Review/Solution.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        
        input.nextLine();
        
       for (int i = 0; i < n; i++) {
           String test = input.nextLine();
           char[] string = test.toCharArray();
           
           for (int j = 0; j < string.length; j++) {
               if (j % 2 == 0) {
                   System.out.print(string[j]);
               }
           }
           
           System.out.print(" ");
           
           for (int j = 0; j < s.length; j++) {
               if (j % 2 != 0) {
                   System.out.print(string[j]);
               }
           }
           System.out.println();
       } 
       
       input.close();
        
    }
}

File: Day 6 - Lets Review/Solution.js
function processData(input) {
    //Enter your code here
    n = input.split('\n')

    for(let i = 1; i < n.length; i++){
        var arr = n[i].split('');

        var even = '';
        var odd = '';

        for(let j = 0; j < arr.length; j++){
            if(j % 2 == 0){
                even = even + arr[j];
            }else{
                odd = odd +arr[j];
            }
        }
        console.log(even + ' '+ odd);
    }
} 

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});
